sydent/db/invite_tokens.sql

```diff
@@ -24,7 +25,8 @@ CREATE TABLE IF NOT EXISTS invite_tokens (
     received_ts bigint, -- When the invite was received by us from the homeserver
     sent_ts bigint, -- When the token was sent by us to the user
     origin_id integer, -- original id in homeserver's DB that this was replicated from (if applicable)
-    origin_server text -- homeserver this was replicated from (if applicable)
+    origin_server text, -- homeserver this was replicated from (if applicable)
+    valid_until_ms integer not null default 0 -- When the invite is due to expire
```

Comment:
Also also: would `NULL` not be a more obvious way to represent 'does not expire' ?

In any case, can you add the 'does not expire' case to the comment?

(Not that it's particularly relevant for sydent, but for postgres, adding a column with a `DEFAULT` requires rewriting the table: https://www.postgresql.org/docs/10/sql-altertable.html#id-1.9.3.33.7)

Reply:
Also also also: none of this should be here, because https://github.com/matrix-org/sydent/blob/30abadf6d3cb2f007ca7a6bbecb1a30ddd91ec76/sydent/db/sqlitedb.py#L37. Take everything I said and apply it to https://github.com/matrix-org/sydent/pull/166/files#diff-422ca0bf77028a3860cd5206bcf74072R151.